src/components/QuestionAnswer.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { ENTER_CHARCODE } from '../utils/Constants';
+
+const QuestionAnswer = React.forwardRef((props, ref) => {
+  const { questionObj, nextQuestion } = props;
+  const [inputVal, setInputVal] = useState('');
+
+  const handleOnClick = () => {
+    nextQuestion();
+  };
+
+  const handleKeypress = (e) => {
+    if (e.charCode === ENTER_CHARCODE) {
+      nextQuestion();
+    }
+  };
+
+  const onChangeHandler = (e) => {
+    setInputVal(e.target.value);
+  };
+
+  useEffect(() => {
+    setInputVal('');
+  }, [questionObj]);
+
+  const exceptThisSymbols = ['e', 'E', '+'];
+  return (
+    <div className="ArithmeticContainer">
+      <div>
+        {questionObj.question}
+        <input
+          ref={ref}
+          type="number"
+          className="Input"
+          value={inputVal}
+          onChange={onChangeHandler}
+          onKeyPress={handleKeypress}
```

Comment:
I think onkeydown should be enough. Also charCode has been deprecated. use key or code for determining enter.

Reply:
Done